src/components/policy/policy_regular/src/sql_pt_representation.cc

```diff
@@ -1342,6 +1342,8 @@ bool SQLPTRepresentation::SaveConsumerFriendlyMessages(
     if (!SaveMessageType(it->first)) {
       return false;
     }
+
+    LOG4CXX_INFO(logger_, "SaveConsumerFriendlyMessages enter");
```

Comment:
@Ypostolov it is not info, looks like trace more.

Reply:
@Ypostolov I think you can remove this log message at all as you already have trace log message for this function